lib/ansible/plugins/inventory/script.py

```diff
@@ -28,6 +28,8 @@
     notes:
         - Enabled in configuration by default.
         - The plugin does not cache results because external inventory scripts are responsible for their own caching.
+        - To write your own inventory script see U(../../../dev_guide/developing_inventory.html#developing-inventory-scripts).
```

Comment:
This should use the R() format for internal xrefs.. .see https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation

Reply:
will that work with relative links? felix suggested using U()

Reply:
well from the docsite perspective, R() with the RST anchor works.  But I lost the plot on what that does for ansible-doc output.

Reply:
IT does nothing in either case, we never got around to defining how to do this .. something i've tried to get defined for many years but had too much controversy. Happy to change to R, its equally useless in console output.

Reply:
@felixfontein - can you pipe in here? I thought R() was preferred but I feel like you're the expert on this... should it stay as U() with a relative link?

Reply:
Well, `R()` is best for the docsite, but it doesn't work with ansible-doc (the command line tool). That one basically ignores the reference part and only shows the reference's title.

In any case, the relative link also doesn't work well with ansible-doc either. You will simply see `../../../dev_guide/developing_inventory.html#developing-inventory-scripts` written out, which isn't helpful either. (I tried by adding `U(../../../dev_guide/developing_inventory.html#developing-inventory-scripts)` to the `description` of my ansible-core's `copy` module and looking at `ansible-doc copy`.)